src/astx/operators.py

```diff
@@ -128,3 +131,67 @@ def get_struct(self, simplified: bool = False) -> ReprStruct:
         key = str(self)
         value = self.value.get_struct(simplified)
         return self._prepare_struct(key, value, simplified)
+
+
+@public
+@typechecked
+class AugAssign(DataType):
+    """AST class for augmented assignment (+= , -= , |=, *= , /= , //= , eg)."""
+
+    OPERATORS = {
+        "+=": operator.iadd,
+        "-=": operator.isub,
+        "*=": operator.imul,
+        "/=": operator.itruediv,
+        "//=": operator.ifloordiv,
+        "%=": operator.imod,
+        "**=": operator.ipow,
+        "&=": operator.iand,
+        "|=": operator.ior,
+        "^=": operator.ixor,
+        "<<=": operator.ilshift,
+        ">>=": operator.irshift,
+    }
+
+    def __init__(
+        self,
+        target: str,
+        aug_op: Literal[
```

Comment:
```suggestion
        op_code: Literal[
```

let's use op_code just to keep consistency with binaryop and unaryop classes  (https://github.com/arxlang/astx/blob/main/src/astx/types/operators.py#L147)

also please create this attributes in the class level as well.
In general it wouldn't be necessary to add the attribute in the class level, but it helps if we want to check the attributes directly in the class. You can create the attributes in the class level for target, op_code, and value.